pyanaconda/ui/common.py

```diff
@@ -640,4 +684,13 @@ def collectCategoriesAndSpokes(paths, klass, displaymode):
     for c in categories:
         ret[c] = collect_spokes(paths["spokes"], c.__name__)
 
+    # As we now have a list of all categories this hub holds we can now register it's controller.
+    # We need the list of categories so that spokes can find out which controller they should use
+    # based on their category.
+    category_names = set()
+    for c in categories:
+        category_names.add(c.__name__)
+
+    controller.add_controller(klass.__name__, category_names)
+
```

Comment:
There is a return statement under it:
```
# As we now have a list of all categories this hub holds we can now register it's controller.
# We need the list of categories so that spokes can find out which controller they should use
# based on their category.
category_names = set()
for c in categories:
    category_names.add(c.__name__)
    
controller.add_controller(klass.__name__, category_names)                                                                                                              
    
return ret
```
I think it's reasonable to have blank line above & bellow this statement as it doesn't really belong to either the for loop above of the return statement bellow. But I guess I can add a comment. :)